packages/@expo/cli/src/run/ios/appleDevice/client/LockdowndClient.ts

```diff
@@ -128,7 +128,6 @@ export class LockdowndClient extends ServiceClient<LockdownProtocolClient> {
       if (resp.EnableSessionSSL) {
         this.protocolClient.socket = new tls.TLSSocket(this.protocolClient.socket, {
           secureContext: tls.createSecureContext({
-            secureProtocol: 'TLSv1_method',
```

Comment:
Add a comment note explaining why we can't add this property back.

Reply:
Done, hope it makes sense.

TL;DR; `secureProtocol` is deprecated in favor of `minVersion` and `maxVersion`, both with proper default values to establish a secure connection based on what both Node and the device support.